src/app/components/home.component.ts

```diff
@@ -578,12 +578,13 @@ export class HomeComponent implements OnInit, AfterViewInit {
     });
 
     // When `watch` folder and `chokidar` detects a file was deleted (can happen when renamed too!)
+    // mark the element in `imageElements[]` as `deleted`
     this.electronService.ipcRenderer.on('single-file-deleted', (event, sourceIndex: number, partialPath: string) => {
       this.imageElementService.imageElements
         .filter((element: ImageElement) => { return element.inputSource == sourceIndex })
         // notice the loosey-goosey comparison! this is because number  ^^  string comparison happening here!
         .forEach((element: ImageElement) => {
-          if (('\\' + partialPath) === path.join(element.partialPath, element.fileName)) {
+          if (partialPath === path.join(element.partialPath, element.fileName)) {
```

Comment:
I'm unsure why I had `\\` before -- I think without it, the functionality didn't work.

But when trying today, it did.

I think my `.vha2` files might have a different path strings. I need to make sure somehow they are normalized.

Perhaps `path.normalize()` 🤔

Reply:
Just going to leave as an `||` for both scenarios 🤷‍♂️ 
Works well-enough 👌